docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -47,19 +47,16 @@ the implementation without compat concerns in future releases.
 - If the `AssemblyVersion` property exists (for example, [Microsoft.CSharp\Directory.Build.props](https://github.com/dotnet/runtime/blob/master/src/libraries/Microsoft.CSharp/Directory.Build.props#L4)), you can change it.
 - If the `AssemblyVersion` property doesn't exist (for example, [System.Runtime\Directory.Build.props](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime/Directory.Build.props)), you can add it.
 
-**If changing the target group**
-- Update both the `Configurations` property in the library's csproj file and the `BuildConfigurations` property in the library's Configurations.props file.
-
 **Update pkg**
  - If changing the target framework
     - Update `SupportedFramework` metadata on the ref ProjectReference to declare the set of concrete platforms you expect your library to support. (see [Specific platform mappings][net-standard table]). Generally will be a combination of netcoreapp2.x, netfx46x, and/or `$(AllXamarinFrameworks)`.
   - If assembly or package version is updated the package index needs to be updated by running
     `dotnet msbuild <Library>/pkg/<Library>.pkgproj /t:UpdatePackageIndex`
 
 **Update tests**
-  - Set `TargetGroup` which will generally match the `TargetGroup` in the src library build configuration. (ex: [System.Runtime\tests\Configurations.props](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/tests/Configurations.props#L3))
+  - Add new `TargetFramework` to the ```TargetFrameworks```.
   - Add new test code following [conventions](project-guidelines.md#code-file-naming-conventions) for new files to that are specific to the new target framework.
-  - To run just the new test configuration run `dotnet msbuild <Library>.csproj /t:RebuildAndTest /p:TargetGroup=<TargetGroup>`
+  - To run just the new test targetFramework run `dotnet msbuild <Library>.csproj /t:RebuildAndTest /p:BuildTargetFramework=<TargetFramework>`
```

Comment:
@Anipik @ericstj so dotnet build -f net472 /t:RebuildAndTest doesn't work with the new config system? What is necessary to make that work? For .NET Framework, the OSGroup will always be Windows_NT, right?

```suggestion
  - To run just the new test targetFramework run `dotnet build <Library>.csproj /t:RebuildAndTest /p:BuildTargetFramework=<TargetFramework>`
```

Reply:
I think explicitly passing in TargetFramework should work (so long as it matches one of the frameworks in TargetFrameworks), shouldn't it @Anipik?  The part that didn't work was the *best* framework detection that used to work before.

Reply:
yes that should work